src/orga/guidelines.rst

```diff
@@ -119,3 +119,25 @@ drops the "main" label and replaces it with "broken". This still makes it unavai
 by default. However, the package remains available so as to avoid the problems listed above.
 
 To get your package relabeled as ``broken`` instead of ``main``, please refer to :ref:`maint_fix_broken_packages`.
+
+
+Becoming a maintainer
+=====================
+
+Conda-forge is a community project and it can therefore happen that feedstocks become temporarily abandoned.
+You can join the maintainer team by following steps:
+
+1) Add your github-id to the ``recipe-maintainers`` section at the bottom of the ``recipe/meta.yaml`` file in the feedstock:
+
+  .. code-block:: yaml
+
+    extra:
+      recipe-maintainers:
+        - current-maintainer
+        - your-github-id
+
+2) Commit and push the change to your fork and open a :term:`PR` against the feedstock you want to become a maintainer of.
+
```

Comment:
Would add re-render the feedstock since this updates the codeowners and maintainer listing in the Readme.

Reply:
Good catch! I added an extra step.

Reply:
Thanks! 😄